geopandas/plotting.py

```diff
@@ -160,9 +166,25 @@ def plot_dataframe(s, column=None, colormap=None,
         scheme : pysal.esda.mapclassify.Map_Classifier
             Choropleth classification schemes
 
+        vmin : float
```

Comment:
I used these for consistency with matplotlib, and thus discoverability by the user (similar to my comment on `ax` vs. `axes` in #208). They use `vmin`/`vmax` for all keyword arguments that allow setting colormap/colorbar limits - see for example [this demo](http://matplotlib.org/examples/pylab_examples/pcolor_demo.html) or the `set_clim()` method in the [colormap module](http://matplotlib.org/api/cm_api.html). I think the abbreviation stands for "minimum/maximum *value*". I agree it's not the best choice (I personally would have chosen `cmin`/`cmax` as well), but since matplotlib is so widely used, IMHO it's better to stay consistent with them than invent our own slightly different naming scheme in an attempt to "fix" this, but in doing so cause frustration for any matplotlib users who expect it to be called `vmin`/`vmax`. (On this note, Joel Spolsky has a good [article](http://www.joelonsoftware.com/uibook/chapters/fog0000000061.html) on consistency which I think it would be wise to follow in this case.)